assets/stylesheets/emergent-paywall.scss

```diff
@@ -0,0 +1,727 @@
+				margin: 10px 0;
+
+				.btn-tile-aggregates {
+					overflow: hidden;
+					height: 70px;
+					margin-bottom: 15px;
+
+					img {
+						object-fit: contain;
+						padding: 3px 1px;
+						margin: 1px;
+						width: 80px;
+						height: 28px;
+					}
+				}
+
+				.btn-tile-footer {
+					font-size: 14px;
+					text-transform: capitalize;
+					font-weight: 500;
+				}
+			}
+
+			&:hover {
+				transition: all 150ms ease-in-out;
+				border: 1px solid $gray-lighten-10;
+			}
+
+		}
+
+
+		@include breakpoint( '<480px' ) {
+			height: auto;
+
+			.btn-tile-cont {
+				flex: 100%;
+				height: auto;
+				border: 1px solid $gray-lighten-20;
+				padding: 4px;
+				margin: 0 auto 25px;
+
+				.btn-tile {
+					.btn-tile-aggregates {
+						max-height: 70px;
+						height: auto;
+						margin-bottom: 10px;
+					}
+
+					.btn-tile-footer {
+						font-size: 16px;
+					}
+				}
+			}
+		}
+
+	}
+
+	.checkbox, div[ng-show='updateAddressCheckbox'] {
+		clear: both;
+		text-align:left;
+		font-size: 14px;
+
+		input {
+			margin-right: 8px;
+			height: 14px;
+		}
+	}
+
+	.content {
+
+		form {
+			margin-bottom: 15px;
+
+			div[simple-template='link-instrument'] {
+				.display-icons {
+					//Move the CC brands icons to the right of the cardNumber field
+					position: absolute;
+					top: 6px;
+					left: 250px;
+
+					.radio-highlight {
+						display: inline-block;
+						padding: 4px;
+
+						&[title='rupayDebit'] {
+							margin-right: 6px;
+						}
+
+						&[title='masterCardDebit'] {
+							margin: 0 -10px;
+						}
+
+						img {
+							height: 20px;
+						}
+
+						@include breakpoint( '<480px' ) {
+							img {
+								height: 10px;
+							}
+						}
+					}
+				}
+			}
+
+			div[simple-template='link-expiration'] {
+				display: inline-flex;
+				margin: 0 20px 5px 0;
+				position: relative;
+				top: -25px;
+
+				// Hide a couple of empty fields on the card form so that the CVV can be next to the expiration
+				~ div[simple-replace='{FIELD:\'expirationMonth\'}'] {
+					display: none;
+				}
+
+				~ div[simple-replace='{FIELD:\'expirationYear\'}'] {
+					display: none;
+				}
+			}
+
+			div[simple-template='link-security-code'] {
+				position: relative;
+				top: -25px;
+				margin-bottom: -5px;
+			}
+
+			div[simple-template='link-security-code'], div[simple-template='link-billToCity-and-billToPostalCode'], div[simple-template='link-province'] {
+				display: inline-flex;
+			}
+
+			.form-group {
+				margin: 5px 0 10px;
+
+				label {
+					display: block;
+					font-size: 14px;
+					font-weight: 600;
+					margin-bottom: 5px;
+
+					// Labels are lowercase.
+					&::first-letter {
+						text-transform: capitalize;
+					}
+				}
+
+
+
+				&.select-inline {
+					select {
+						display: inline-block;
+						min-width:100px;
+					}
+				}
+
+				.form-control {
+					margin: 0;
+					padding: 7px 14px;
+					color: $gray-dark;
+					font-size: 16px;
+					line-height: 1.5;
+					border: 1px solid #c8d7e1;
```

Comment:
Any reason why hex values are being used for color? Ideally, there shouldn't be a need to use anything but the sass variable.